app/assets/javascripts/actions/task.js

```diff
@@ -0,0 +1,46 @@
+import actionTypes from './actionTypes';
+
+export const taskAdd = (task, story) => ({
+  type: actionTypes.ADD_TASK,
+  task,
+  story
+});
+
+export const removeTask = (task, story) => ({
+  type: actionTypes.REMOVE_TASK,
+  task,
+  story
+});
+
+export const updateTask = (task, story) => ({
+  type: actionTypes.UPDATE_TASK,
+  task,
+  story
+});
+
+export const addTask = (projectId, story, newAttributes) => {
```

Comment:
What do you think about changing the name of these methods? Because we'd have `taskAdd` and `addTask`, `removeTask` and `deleteTask`, etc. and it might be difficult to differentiate one from the other